### PR TITLE
Add org.sugarlabs.falabracman

### PR DESCRIPTION
**Pull request by SudoSu-bham**
### Please confirm your submission meets all the criteria

- [ x] Please describe the application briefly.
It is a Sugar activity, it helps in learning some words, by collecting all the letters in the right order and by avoiding the lake otherwise you will fall into it.

- [ x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [ x] I have read the and followed all the [Submission and licence requirements][reqs].
- [ x] I have [built][build] and tested the submission locally.
- [ x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.sugarlabs.Falabracman

**Comment by bbhtt** [comment]
cc @tchx84

File: falabracman-info.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ show_launcher = yes
+ repository = https://github.com/sugarlabs/falabracman-activity.git
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
++developer_id = org.sugarlabs
++tags = Education
++url = https://github.com/sugarlabs/falabracman-activity
++summary = Sugar activity to learn words
++description =  It helps in learning some words, by collecting all the letters in the right order and by avoiding the lake otherwise you will fall into it.
++screenshots = https://imgur.com/a/597YvUB https://imgur.com/a/weEN2rH https://imgur.com/a/2iuXSMr
```
Comment (bbhtt) [inline falabracman-info.patch:27]: Why are this on imgur?

File: org.sugarlabs.Falabracman.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            ],
+            "build-options": {
+                "env": {
+                    "PORTMIDI_INC_PORTTIME": "1",
+                    "PYGAME_EXTRA_BASE": "/app"
+                },
+                "arch": {
+                    "aarch64": {
+                        "env": {
+                            "ORIGLIBDIRS": "/lib:/lib64:/lib/aarch64-linux-gnu"
```
Comment (bbhtt) [inline org.sugarlabs.Falabracman.json:44]: Are these correct? Shouldn't it be `/app/lib` etc.

File: org.sugarlabs.Falabracman.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                    "path": "falabracman-info.patch"
+                }
+            ],
+            "post-install": [
+                "sugarapp-gen-mimetypes activity/activity.info mimetypes",
+                "sugarapp-gen-appdata activity/activity.info appdata",
+                "sugarapp-gen-desktop activity/activity.info desktop --mimetypes mimetypes",
+                "install -D mimetypes /app/share/mime/packages/org.sugarlabs.Falabracman.xml",
+                "install -D mimetypes /app/share/sugar/activities/Falabracman.activity/activity/mimetypes.xml",
+                "install -D appdata /app/share/metainfo/org.sugarlabs.Falabracman.appdata.xml",
```
Comment (bbhtt) [inline org.sugarlabs.Falabracman.json:86]:
```suggestion
                "install -D appdata /app/share/metainfo/org.sugarlabs.Falabracman.metainfo.xml",
```

File: falabracman-info.patch (hunk 1 above)
Comment (tchx84) [reply falabracman-info.patch:27]: No particular reason, we've been using that service for other apps under org.sugarlabs. If you think we're better served by something else, please suggest :)

Comment (bbhtt) [reply falabracman-info.patch:27]: images on imgur tend to get lost, if upstream is ok with it I don't mind.

File: org.sugarlabs.Falabracman.json (hunk 2 above)
Comment (SudoSu-bham) [reply org.sugarlabs.Falabracman.json:44]: If the application or build process needs to access system-wide libraries than the above configuration is good.

our [Pippy](https://github.com/flathub/org.sugarlabs.Pippy/blob/master/org.sugarlabs.Pippy.json) and [Physics](https://github.com/flathub/org.sugarlabs.Physics/blob/master/org.sugarlabs.Physics.json) are also using the same configurations

**Comment by bbhtt** [comment]
Were screenshots fixed? That was the build faliure

File: falabracman-info.patch (hunk 1 above)
Comment (SudoSu-bham) [reply falabracman-info.patch:27]: If I am using images link which are uploaded on github than I am getting this error
`appstream-screenshots-not-mirrored-in-ostree`. May you suggest best way to include image which is better than imgur and has no problem with flatpak.

Comment (bbhtt) [reply falabracman-info.patch:27]: You can ignore that. For imgur images it has to be a direct link eg. https://i.imgur.com/NS20nXf.png

Same for Github it has to be the raw.githubusercontent.com link.

**Comment by SudoSu-bham** [comment]
> Were screenshots fixed? That was the build faliure

Yes, May you please review it

**Comment by bbhtt** [comment]
bot, build org.sugarlabs.Falabracman

**Pull request closed without merging**